src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -391,9 +391,9 @@ examples below to get you started.
 #### Using ChromeDriver
 
 [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) 2.3.0
-supports Chrome 59 and later and works with headless Chrome. In some cases, you may need
-Chrome 60 to work around bugs. For example, there are known issues with taking
-screenshots in Chrome 59.
+was the first version to support Chrome 59 and work with headless Chrome,
+but you'll want Chrome 61 to work around screenshot bugs. The good news
+is that more recent versions of ChromeDriver (2.3.2) work with Chrome 61!
```

Comment:
Thanks for the change! It's a lot less intimidating to use ChromeDriver with headless Chrome now.

I'm a bit confused by this paragraph. It says ChromeDriver 2.3.0 "supports" Chrome 59, but needs Chrome 61 because of some bugs, and then says there's this new ChromeDriver 2.3.2 which also works with Chrome 61.

IMHO it'd be much clearer if we only mention the latest and working versions, especially since 61 is stable now. Something like:

> [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) supports headless Chrome. ChromeDriver 2.3.2 and Chrome 61 are the minimum versions required because there were screenshot bugs in older versions.

(Of course the wording could use some improvement!)

Reply:
Yea, I already changed it to this:

`[ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) 2.32 uses Chrome 61 and works well with headless Chrome.`